### PR TITLE
Revert "make nuget workflow easy to debug. (#15693)"

### PR DESCRIPTION
**Pull request by snnn**
### Description
This reverts commit 53ff50d19a8a49e45fba33791acaf5eee5cbaeaa because it make the nuget pipeline fail.

Welcome to modify the changes and resubmit them. For now I need to revert  the commit to make our pipelines green.

### CODE REVIEW

**Review by jchen351 — DISMISSED** [review]

File: csharp/test/Microsoft.ML.OnnxRuntime.Tests.NetCoreApp/InferenceTest.netcore.cs [hunk 1]
```diff
@@ -928,51 +846,52 @@ private void TestRegisterCustomOpLibraryV2()
...
+                            )));
 
-        [SkipNonPackageTests(DisplayName = "TestRegisterCustomOpsWithFunction")]
-        private void TestRegisterCustomOpsWithFunction()
-        {
-            using (var option = new SessionOptions())
-            {
-                try
-                {
-                    Marshal.Prelink(typeof(CustomOpLibrary).GetMethod("RegisterCustomOpsAltName"));
```
Comment (snnn) [inline csharp/test/Microsoft.ML.OnnxRuntime.Tests.NetCoreApp/InferenceTest.netcore.cs:959]: For unknown reason, this "RegisterCustomOpsAltName " method cannot be found.

**Comment by skottmckay** [comment]
Please do not revert https://github.com/microsoft/onnxruntime/commit/b9ef93d1305daa24d434effbbca48787add82567

It is required for BUILD demos. I'd rather disable the test temporarily.

File: csharp/test/Microsoft.ML.OnnxRuntime.Tests.NetCoreApp/InferenceTest.netcore.cs (hunk 1 above)
Comment (skottmckay) [reply csharp/test/Microsoft.ML.OnnxRuntime.Tests.NetCoreApp/InferenceTest.netcore.cs:959]: The symbol should be in the custom ops library but possibly Marshal.Prelink works differently on Linux. The test isn't essential as we're calling RegisterCustomOps directly for extensions so it's a different path.

**Comment by snnn** [comment]
> Please do not revert [b9ef93d](https://github.com/microsoft/onnxruntime/commit/b9ef93d1305daa24d434effbbca48787add82567)
> 
> It is required for BUILD demos. I'd rather disable the test temporarily.

You can create a PR and submit it now. We do not cut the branch today. There are still a few days left. There are some other people working on pipelines. If I don't bring the pipelines back to normal, they cannot validate their changes. They also are preparing their work for BUILD demos. They have the same deadline.

**Comment by skottmckay** [comment]
#15747 will address the C# unit test failure short term

**Review by skottmckay — APPROVED** [review]

**Pull request merged**